cmd/kube-apiserver/app/aggregator.go

```diff
@@ -290,9 +290,10 @@ var apiVersionPriorities = map[schema.GroupVersion]priority{
 }
 
 func apiServicesToRegister(delegateAPIServer genericapiserver.DelegationTarget, registration autoregister.AutoAPIServiceRegistration) []*v1.APIService {
-	apiServices := []*v1.APIService{}
+	paths := delegateAPIServer.ListedPaths()
+	apiServices := make([]*v1.APIService, 0, len(paths))
 
-	for _, curr := range delegateAPIServer.ListedPaths() {
```

Comment:
same here, I think the existing style is cleaner, and this is only executed once at process startup so the re-allocation of the slice is negligible